src/RLinearAlgebra.jl

```diff
@@ -23,6 +23,9 @@ module RLinearAlgebra
 
 using LinearAlgebra, Random, Distributions
 
+import SparseArrays: sprandn, SparseMatrixCSC
+
+import Hadamard: hadamard
```

Comment:
I am not sure where this function is used in your code.

Reply:
The new code should clarify the use of these functions.